forcedotcom/commerce-on-lightning
# [@W-12185041@] install retrigger-place-order flow

### What does this PR do?
Deploy the retrigger-place-order flow for B2B and B2C store

### What issues does this PR fix or reference?
@W-12185041@

### Functionality Before


### Functionality After
The retrigger-place-order flow will be automatically installed for b2b and b2c store

### How to Test/Testing Effort 
After the store is created, the Retrigger Place Order flow should be installed in Setup/Flow
